presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -220,6 +220,16 @@ else if (controlResult.getState() != State.SUCCESS) {
             }
         }
 
+        // Re-run the test query to confirm that the results don't match, in case the underlying data changed, but was cached on the test tier.
```

Comment:
What exactly does "in case the underlying data changed, but was cached ..." refer to? We only cache metadata.